src/lib.rs

```diff
@@ -205,14 +209,14 @@ impl<T: Ord> IntervalHeap<T> {
     ///
     /// # Examples
     ///
-    /// ```rust
+    /// ```
     /// # extern crate "interval-heap" as interval_heap;
     /// # fn main() {
     /// use interval_heap::IntervalHeap;
     ///
     /// let heap = IntervalHeap::from_vec(vec![5, 1, 6, 4]);
     /// assert_eq!(heap.len(), 4);
-    /// assert_eq!(heap.get_min_max(), Some((&1, &6)));
+    /// assert_eq!(heap.min_max(), Some((&1, &6)));
```

Comment:
One more letter and we can get some sweet tree search algorithms in this repo :dancer: